dev/io.openliberty.wsoc.2.1.internal_fat/fat/src/io/openliberty/wsoc/tests/Basic21Test.java

```diff
@@ -208,22 +208,22 @@ public void testSSCZeroTimeOut() throws Exception {
 
     @Mode(TestMode.FULL)
     @Test
-    public void testSSCNegativeoTimeOut() throws Exception {
-        this.runAsLSAndVerifyResponse("TimeOutTest", "testNegativeoTimeOut");
+    public void testSSCNegativeTimeOut() throws Exception {
+        this.runAsLSAndVerifyResponse("TimeOutTest", "testNegativeTimeOut");
         String result  = LS.waitForStringInTraceUsingMark("Session timeout -12 is less than 1. No timeout enabled");
         assertNotNull("Timeout message not found!", result);
     }
 
     @Mode(TestMode.LITE)
     @Test
-    public void testUserPropertiesOnServer() throws Exception {
-        userprop.testUserPropertiesOnServer();
+    public void testSSCUserPropertiesOnServer() throws Exception {
+        this.runAsLSAndVerifyResponse("UserPropertiesTest", "testUserPropertiesOnServer");
     }
 
     @Mode(TestMode.LITE)
     @Test
-    public void testSSCUserPropertiesOnServer() throws Exception {
-        this.runAsLSAndVerifyResponse("UserPropertiesTest", "testUserPropertiesOnServer");
+    public void testUserPropertiesOnServer() throws Exception {
```

Comment:
So, the order of the test cases isn't consistent between JDKs. So if this fixes it on one JDK  the chances are likely that it will still fail on another JDK. Is it possible to just combine these two tests into one `@Test` method to ensure proper ordering?

Reply:
In addition are these new tests for 2.1?

Reply:
My hope it to see if there's any improvement and if not, I can try another fix for this. I can't reproduce this jetty timeout locally, so I'm not sure how it's triggered with it.  Unless it's just slow hardware.